pkg/apiserver/plugins.go

```diff
@@ -58,6 +60,7 @@ var (
 		shootstatedeletionvalidator.PluginName,     // ShootStateDeletionValidator
 		customverbauthorizer.PluginName,            // CustomVerbAuthorizer
 		shootvpa.PluginName,                        // ShootVPAEnabledByDefault
+		managedseedvalidator.PluginName,            // ManagedSeed
```

Comment:
Shouldn't the `shootmanagedseed.PluginName` be in this list as well?

Reply:
Yes, it was "lost" during the rebase I did 2 hours ago, thanks for noticing.

Reply:
Done